src/api.js

```diff
@@ -91,13 +104,15 @@ export default class Api {
    * TODO: If more than X results (default is 25 on server), split in several
    * calls. Related: https://github.com/mozilla-services/cliquet/issues/318
    *
+   * @param  {String} bucketName   The bucket name.
    * @param  {String} collName The collection name.
    * @param  {Array}  records  The list of record updates to send.
-   * @param  {Object} headers  Headers to attach to each update request.
    * @param  {Object} options  Options.
    * @return {Promise}
    */
-  batch(collName, records, headers={}, options={safe: true}) {
+  batch(bucketName, collName, records, options={headers: {}, safe: true}) {
```

Comment:
I'm now wondering if `bucketName` shouldn't be an option; users may be most likely wanting the default bucket most of the time… What do you think?